README.md

```diff
@@ -18,7 +18,7 @@ Be sure that you have installed a GIT client, Node.JS (latest or LTS). You can u
 
 ### Using as a template
 
```

Comment:
This section doesn't make sense in your repository. It would be best to remove.

Reply:
done